README.md

```diff
@@ -68,6 +68,17 @@ Open the index page in a web browser
     android: ./build/processor-sdk-android-<FAMILY>/esd/docs/[version]/index.html
     debian: ./build/processor-sdk-debian-<FAMILY>/esd/docs/[version]/index.html
 
+## Live Preview on GitHub Pages
+
+GitHub Pages are now live for all `DEVFAMILY` and `OS` supported by this repository.
+This means that for every pull request merged into the master branch, an equivalent
+preview will be available on GitHub Pages immediately.
+
+You can access the latest bleeding-edge documentation at the following link:
+    - [Processor SDK Documentation](https://texasinstruments.github.io/processor-sdk-doc/)
```

Comment:
```suggestion

    - [Processor SDK Documentation](https://texasinstruments.github.io/processor-sdk-doc/)
```
Forgot that markdown requires newlines around lists more so than rst.

Reply:
@StaticRocket My intention was to have it like psdk-docker link  - https://github.com/TexasInstruments/processor-sdk-doc?tab=readme-ov-file#install-tools-on-ubuntu 

Was not looking to have it as a list since it's anyway a single URL

Reply:
https://github.com/TexasInstruments/processor-sdk-doc/pull/118/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 is the current markdown preview which looks similar to what I was anticipating.

Any reasons we should have it as a list?

Reply:
Oh, if you didn't intend it to be a list then it's fine. I meant for container link to be a list when I first did it, but I goofed. Assumed you were trying to do the same.

Reply:
Ahh nope. I believe list isn't required here as for both the cases as it's a single entry. Even the psdk-doc-docker link looks fine atleast to me.